docs/cni/kubernetes/AWS.md

```diff
@@ -212,10 +212,15 @@ aws ec2 run-instances \
   --key-name mykey \
   --security-group-ids $SECURITY_GROUP_ID \
   --user-data file://cloud-config/node-config.yaml
-#  --subnet $SUBNET_ID 
+#  --subnet $SUBNET_ID
 #  Include the subnet param above if using a non-default VPC
 ```
 
+Finally, disable `Source/Dest. Check` on each instance (including the master) to allow routing between pods without needing IP in IP.  All instances must be in the same subnet.  You can do this with the CLI, or in the `Networking` part of the instances' right click menus.
+```
+aws ec2 modify-instance-attribute --instance-id $INSTANCE_ID --source-dest-check "{\"Value\": false}"
```

Comment:
The `$INSTANCE_ID` environment variables are not defined in this doc.  You should add these like you did in the  `docs/AWS.md` file and probably modify this command in a similar way to have each necessary `aws cli` call.